vane-portals/src/main/java/org/oddlama/vane/portals/Portals.java

```diff
@@ -166,6 +167,22 @@
 	)
 	public ExtendedMaterial config_default_icon;
 
+	@ConfigDouble(
+		def = 0.9,
+		min = 0.0,
+		max = 2.0,
```

Comment:
I don't actually know what the range for sound effects is, is this an accurate max?

Reply:
Technically, it's unlimited. But due to the way sounds work in minecraft, anything `> 1.0` will not actually be louder, but instead will be heard from further away. Therefore it's actually nonsense that I'm using `1.2` for one of the defaults :P

The actual volume is linearly calculated based on the distance from the sound. 16 blocks is the default range (`volume = 1.0`). The volume parameter inversly scales the apparent distance. So any volume `< 1` will work as expected, but any volume > 1.0 just causes it to be heard from further away. So volume = 2.0 would make the sound be heard from 0-32 blocks distance instead of 0-16, but still with a linear falloff from 100% to 0% real volume.

I'd probably suggest setting the max to 1.0 because nobody will expect it to increase range instead of getting louder. Do you have any opinions?

Reply:
1.0 makes sense to me, in the case